gendox-core-api/src/main/java/dev/ctrlspace/gendox/gendoxcoreapi/controller/OrganizationController.java

```diff
@@ -83,12 +104,14 @@ public Organization updateOrganization(@PathVariable UUID id, @RequestBody Organ
 
 
     // TODO: Has Permission OP_DELETE_ORGANIZATION
+    @PreAuthorize("@securityUtils.hasAuthorityToRequestedOrgId('OP_DELETE_ORGANIZATION')")
     @DeleteMapping("/organizations/{id}")
     @ResponseStatus(HttpStatus.NO_CONTENT)
-    public void deleteOrganization(@PathVariable UUID id) throws Exception{
+    public void deleteOrganization(@PathVariable UUID id, @Valid OrganizationCriteria critetia) throws Exception {
```

Comment:
This `@Valid OrganizationCriteria critetia` shouldn't be here. The Delete by ID API doesn't get params.
We need to find another way to do the `@PreAuthorize`.
Please remove this and create a ticket on me to fix the preauthorize.